atd-vze/src/Components/GridExportData.js

```diff
@@ -133,6 +137,14 @@ const GridExportData = ({
         }
       });
 
+      columnsToEscapeDoubleQuotes.forEach(col => {
```

Comment:
this approach looks sound to me, but isn't it possible we end up with quotes in other fields? might be better to just escape any field that has a string value? it will be a performance hit for sure, although you could also test if a value contains a quote before escaping it (iirc regex is quite expensive but a string-contains check not so much).

in any case, i was surprised to see we were already using a CSV library and that it was not handling quote escaping. fwiw there are [open PRs](https://github.com/react-csv/react-csv/issues/199) 🤷 

i was also wondering about the need to escape commas as well, but apparently the default behavior is to wrap all values [in double quotes](https://github.com/react-csv/react-csv#--enclosingcharacter-props), which takes care of commas.

@roseeichelmann is there

Reply:
one more idea on the topic of just quote-escaping every field... you could generate the list of string-type columns dynamically up front by check the `typeof` for the first not-null value of every column in the dataset 🤔

Reply:
yeah i was wondering about escaping all string columns too, i was looking into the other columns we have and came to the conclusion that the only other columns that might have double quotes are in the `columnsToClean`list and those cases were already handled by removing the double quotes in the loop frank commented. im not sure whats the best approach but it does make sense to me to quote escape every string field to cover all our bases and make sure no fields w/ double quotes are flying under our radar and throwing off the formatting. i was thinking of the performance hit but maybe its negligible!